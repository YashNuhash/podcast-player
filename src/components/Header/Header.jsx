import React from 'react'

import SearchBar from '../SearchBar/SearchBar'
import styles from './Header.module.scss'

export const Header = () => (
  <header className={styles.Header}>
    <SearchBar />
  </header>
)

export default Header
