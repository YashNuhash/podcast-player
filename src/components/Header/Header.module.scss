@import '../../utils';

.Header {
    margin-bottom: 6rem;
}
