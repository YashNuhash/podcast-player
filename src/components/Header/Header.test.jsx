import React from 'react'
import { render } from '@testing-library/react'
import { Header } from './Header'

it('should render Header component', () => {
    const { asFragment } = render(<Header />)
    expect(asFragment()).toMatchSnapshot()
})
