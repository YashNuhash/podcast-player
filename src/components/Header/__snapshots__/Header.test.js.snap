// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component 1`] = `
<header
  className="Header"
>
  <SearchBar />
</header>
`;
